templates/strings-dot-syntax-swift3.stencil

```diff
@@ -10,12 +10,14 @@ import Foundation
 // swiftlint:disable nesting
 // swiftlint:disable variable_name
 // swiftlint:disable valid_docs
+{% macro parametersBlock params %}{% for type in params.types %}_ p{{forloop.counter}}: {{type}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
```

Comment:
Wondering what it would like (i.e. would it be more readable or less readable?) if you used a `MapNode` here instead?